src/components/BookCard/BookCardMain.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import BookCardBody from './BookCardBody.jsx';
+import BookCardHeader from './BookCardHeader.jsx';
+import styles from './BookCardMain.module.css';
+
+export default function BookCardMain({
+	coverImg,
+	title,
+	publishDate,
+	quote,
+}) {
+	return (
+		<div className={styles.bookCard}>
+			<BookCardHeader coverImg={coverImg} />
```

Comment:
I don't think the header and body should be split into components. Are we going to re-use those components somewhere else? I would suspect not.

In this case, splitting this component up makes it difficult to modify. Where do I need to write CSS to adjust how something looks? Which component contains a particular piece of text in the card? I think splitting this component up makes it harder to understand for other people reading your code :)